docs/get-started/installing-cardano-node.md

```diff
@@ -83,7 +83,13 @@ Use the following command to install `ghcup`
 ```bash
 curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh
 ```
-Please follow the instructions and provide the necessary input to the installer. Once complete, you should have `ghc` and `cabal` installed to your system.
+Please follow the instructions and provide the necessary input to the installer.\
+`Do you want ghcup to automatically add the required PATH variable to "/root/.bashrc"?` - (P or enter)\
```

Comment:
I'd never recommend use of root

Reply:
Good catch, I will update it, in my defense, I was just testing this on my test VM, what is already destroyed. :)